src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -6,41 +6,60 @@ import {
   RefreshResult,
   SelfServeBaseClass,
   SelfServeNotification,
+  SelfServeNotificationType,
   SmartUiInput,
 } from "../SelfServeTypes";
-import { refreshDedicatedGatewayProvisioning } from "./SqlX.rp";
+import {
+  ResourceStatus,
+  refreshDedicatedGatewayProvisioning,
+  updateDedicatedGatewayResource,
+  deleteDedicatedGatewayResource,
+  getCurrentProvisioningState,
+} from "./SqlX.rp";
+
+let disableAttributesOnDedicatedGatewayChange = false;
```

Comment:
this variable will never be changed in the context of the sqlx class, might as well set it as false

Reply:
Why? This does change, basically this variable controls the following:

disable attributes if "dedicated gateway" is enabled = true;

if not 

disable attributes if "dedicated gateway" is enabled = false;

Reply:
Will store response in the current state map